packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -34,11 +34,14 @@ import {
 } from './ChatAdapter';
 import { AdapterError } from '../../common/adapters';
 /* @conditional-compile-remove(file-sharing) */
-import { FileUploadAdapter, convertFileUploadsUiStateToMessageMetadata } from './AzureCommunicationFileUploadAdapter';
-/* @conditional-compile-remove(file-sharing) */
-import { AzureCommunicationFileUploadAdapter } from './AzureCommunicationFileUploadAdapter';
+import {
+  FileUploadAdapter,
+  AzureCommunicationFileUploadAdapter,
```

Comment:
Is /* @conditional-compile-remove(file-sharing) */ or /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ missed here?